### PR TITLE
Adding metrics for geneve tunnel interface

### PR DESCRIPTION
**Pull request by Pardhakeswar**
Signed-off-by: Pardhakeswar Pacha <ppacha@nvidia.com>

### CODE REVIEW

File: go-controller/pkg/metrics/ovs.go [hunk 1]
```diff
@@ -684,6 +695,64 @@ func setOvsInterfaceStatusFields(interfaceBridge, interfacePort, interfaceName,
...
+	}
+}
+
+// geneveInterfaceMetricsUpdate updates the geneve interface
+// metrics obtained through netlink library equivalent to
+// (ip -s li show genev_sys_6081)
+func geneveInterfaceMetricsUpdate() (err error) {
+
+	defer func() {
+		if r := recover(); r != nil {
```
Comment (dcbw) [inline go-controller/pkg/metrics/ovs.go:734]: @Pardhakeswar why do we need a recover here? what situations would cause a panic?

Comment (girishmg) [reply go-controller/pkg/metrics/ovs.go:734]: yeah, we don't need it in this context.

Comment (Pardhakeswar) [reply go-controller/pkg/metrics/ovs.go]: @dcbw @girishmg updated the PR by removing the recover code.

**Pull request merged**